test/bootchooser.c

```diff
@@ -166,7 +166,7 @@ bootstate.system1.priority=20\n\
 }
 
 static void bootchooser_grub(BootchooserFixture *fixture,
-		gconstpointer user_data)
+                             gconstpointer user_data)
```

Comment:
Mh, doesn't this change *all* occurrences of multi-line function parameter definitions? But I also couldn't find any pattern for this in the code we had. Just notice that it is different from function call parameter alignment, but that might be ok.